.travis.yml

```diff
@@ -24,9 +24,10 @@ before_install:
   - sudo apt-get update
   - |
     sudo apt-get install -y rsync python3 \
-     python-ply  python-pytest  python-pytest-catchlog  python-mock  python-pytest-mock \
+    python-ply  python-pytest  python-pytest-catchlog  python-mock  python-pytest-mock \
```

Comment:
I think Chris was aligning the lines on the '-' so we can see we have the same things installed for python2 and python3. Can you keep to that please?

Reply:
Ok changed, so requirements are now in aligned